server/src/main/java/org/elasticsearch/search/aggregations/metrics/AggregatorReducer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.search.aggregations.metrics;
```

Comment:
maybe move this to the `org.elasticsearch.search.aggregations` package? Since it is also used by bucket aggregations?

Reply:
I just realised about this myself. I will do, I started with metrics and forgot to move the interface.

Reply:
done